src/DynamoCore/Configuration/GraphChecksumItem.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.ObjectModel;
+using Dynamo.Core;
+using Dynamo.Properties;
+
+namespace Dynamo.Configuration
+{
+    /// <summary>
+    /// Represents the stringified version of the nodes connections from a graph
+    /// </summary>
+    public class GraphChecksumItem : NotificationObject
```

Comment:
why have you inherited from NotificationObject and used `RaisePropertyChanged` this is only used for binding to WPF view layer - are you doing that?

Reply:
Fixed.